app/assets/src/components/common/Header.jsx

```diff
@@ -261,15 +262,15 @@ const UserMenuDropDown = ({
       <BareDropdown.Item
         key="help"
         text={
-          <ExternalLink
+          <a
```

Comment:
We still want to use ExternalLink... you should be able to use the classname `option` (passed below) to apply the style you need.

Reply:
@tfrcarvalho I'm not sure what you mean by being able to use the classname option to apply the style I need. Do you mean adding in a `color` property to cs.option?

Reply:
yes, for color you might need to use the `!important` ( as `color: $black !important;`) due to a legacy css.

Reply:
I'm reading more about the Classnames package and see that `cs.link` in `Link.jsx` component is overwriting the color to be` $primary-light` (blue)

Reply:
Yes, that is the default we want to enforce for all links. In this case, it is an exception and you want to overwrite that.

Reply:
> We still want to use ExternalLink... you should be able to use the classname `option` (passed below) to apply the style you need.

Just wondering, why do we still want to use ExternalLink?